src/glue/jobs/s3_to_json_s3.py

```diff
@@ -150,6 +150,8 @@ def update_json_schemas(s3_obj, archive_map, json_schemas):
         self_ref_schema_list = get_self_ref_schema_list(z)
         for json_path in contents:
             file_name = os.path.basename(json_path)
+            if file_name == "microphone.json":
```

Comment:
Could there be a comment somewhere briefly explaining this change?

Reply:
I think a more effective method of tracking the reasons behind changes is with git blame. Anyone can see that this change is associated with this PR, and can track the Jira ticket from there.

Reply:
Yes I like that, I remember seeing that before.